### PR TITLE
New design, chapter hierarchy, design system integration

### PR DESCRIPTION
**Pull request by kylesuss**
The new design & guide hierarchy for Learn Storybook 🎉 

@domyen todos:

- [x] Remove the [extra guides I added](https://github.com/chromaui/learnstorybook.com/blob/next/src/components/screens/IndexScreen/Guides.js#L76-L134) in the IndexScreen/Guides component
- [x] Change the og:image, site meta (description, title, etc.)
- [x] Add site meta (description, title, og:) to /team and chapters @kylesuss 
- [x] Update OG images for guides and chapters (see [commit](https://github.com/chromaui/learnstorybook.com/pull/106/commits/fc2f78c594c06cbf741510955d8f83d4a43509f9)) @kylesuss 
- [x] Get logos for remaining sponsors (if any), add them
- [x] Update logos on the index testimonials section
- [x] Make sure subheadline for testimonial users is appropriate (I just put the company Kyle works for)
- [x] Add the 2nd and 3rd testimonial
- [x] Test subscribing to the newsletter
- [x] Validate that the author, contributor count & contributor selection is appropriate on the intro to storybook guide page
- [x] Add the `heroDescription` and `overview` frontmatter copy to the [`intro-to-storybook` guide md file](https://github.com/chromaui/learnstorybook.com/blob/next/content/intro-to-storybook/index.md)

### CODE REVIEW

File: src/components/composite/Header.js [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+  }
+`;
+
+const MobileMenuNavItem = styled(NavItem)`
+  order: 1;
+  margin-right: 10px;
+
+  > span: focus {
+    outline: none;
+  }
```
Comment (kylesuss) [inline src/components/composite/Header.js]: @domyen heads up ☝️ -- I removed this for consistency w/ the old style. Had to make an assumption for now. Feel free to remove if you are ok w/ the button focus.

**Comment by leerob** [comment]
Just pulled this down and ran it - looks AMAZING! 😍 

Let me know if there's any way to help once things are broken out a bit more.

File: content/intro-to-storybook/index.md [hunk 2]
```diff
@@ -27,20 +27,20 @@ authors:
...
       name: "Dominic Nguyen",
-      detail: "Storybook designer",
+      detail: "Storybook design",
     },
   ]
 contributors:
   [
     {
       src: "https://avatars2.githubusercontent.com/u/22988955",
       name: "Jonnie Bigodes",
```
Comment (jonniebigodes) [inline content/intro-to-storybook/index.md]: @domyen i was going to modify this, but it slipped my mind. Can you please change `Jonnie Bigodes` to João Cardoso? That's my actual name.  The username "jonniebigodes  or "Jonnie bigodes" is a nickname that was given to me when i was a kid  and kinda stucked 😄

**Comment by domyen** [comment]
For sure! I'll update. Mind linking your GH avatar url if you have one?

File: content/design-systems-for-developers/react/en/architecture.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+Following this method, we end up with UI primitives: Avatar, Badge, Button, Checkbox, Input, Radio, Select, Textarea, Highlight (for code), Icon, Link, Tooltip, and more. These building blocks are configured in different ways to assemble countless unique features and layouts in our client apps.
+
+![Variants in one component](/design-systems-for-developers/design-system-consolidate-into-one-button.jpg)
+
+Add the components we found by downloading them to your computer and dropping them into your repository, as well as removing the application files that Create React App provided:
+
+```
+rm -rf src/*
+
+svn export [https://github.com/chromaui/learnstorybook-design-system/branches/download-1(https://github.com/chromaui/learnstorybook-design-system/branches/download-1/src)</h6>
```
Comment (kylesuss) [inline content/design-systems-for-developers/react/en/architecture.md]: @domyen heads up this line looks funky

File: content/design-systems-for-developers/react/en/architecture.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+In addition to UI components, it makes sense to include styling constants for typography, colors, spacing, etc that are reused across projects. In design system nomenclature global style variables are called “design tokens”. We won’t dive into the theory behind design tokens in this guide, but you can learn more online (here’s a [good article](https://medium.com/eightshapes-llc/tokens-in-design-systems-25dd82d58421)).
+
+Download our design tokens here and add them to your repository.
+
+```
+# Or download from https://github.com/chromaui/learnstorybook-design-system/tree/download-1/src/shared
+
+# and place in the src/shared folder
+
+svn export [https://github.com/chromaui/learnstorybook-design-system/branches/download-1(https://github.com/chromaui/learnstorybook-design-system/branches/download-1/src)/shared src/shared</h6>
```
Comment (kylesuss) [inline content/design-systems-for-developers/react/en/architecture.md]: Here too?

**Pull request merged**